openmm-system-parity/create-expected-changes.py

```diff
@@ -0,0 +1,75 @@
+import json
+from pathlib import Path
+
+import click
+
+from interchange_regression_utilities.models import ExpectedValueChange
+
+
+def create_toolkit_0_10_x_to_0_11_x_changes():
+    """
+    PR #1198 introduces an intentional behavior change in order to follow the SMIRNOFF
+    specification. Old versions (0.10.3 and older) of the OpenFF Toolkit did _not_ aply
+    a switching function, but new versions (0.11.0 and newer) are intended to.
+    """
+
+    return [
+        ExpectedValueChange(
+            openmm_path="Forces/NonbondedForce/useSwitchingFunction",
+            old_value=0,
+            new_value=1,
+        ),
+        ExpectedValueChange(
+            openmm_path="Forces/NonbondedForce/switchingDistance",
+            old_value=-1.0,
+            new_value=0.8,
+        ),
+    ]
+
+
+def create_toolkit_0_11_x_to_interchange_0_2_xchanges():
+    """
+    PR #1198 introduces an intentional behavior change in order to follow the SMIRNOFF
+    specification. Old versions (0.10.3 and older) of the OpenFF Toolkit did _not_ aply
+    a switching function, but new versions (0.11.0 and newer) are intended to.
+    """
+
+    return [
+        ExpectedValueChange(
+            openmm_path="Forces/NonbondedForce/useSwitchingFunction",
+            old_value=1,
+            new_value=0,
+        ),
+        ExpectedValueChange(
+            openmm_path="Forces/NonbondedForce/switchingDistance",
+            old_value=0.8,
+            new_value=-1,
+        ),
+    ]
```

Comment:
@mattwthompson I assumed interchange just hadn't added support yet (but would before the 0.2.0) and so added this to stop the tests complaining until it was added, but it [seems it was an actual bug](https://github.com/openforcefield/openff-interchange/pull/433)?

Reply:
Correct, that was a bug and it was fixed recently. Just to make sure I'm following - the expectation is that Interchange at release time is expected to apply the switching function as per the SMIRNOFF spec, and this code was there to suppress the warning in the mean time, until it was behaving correctly? If so, I will remove this part of the function since toolkit 0.11.x and Interchange 0.2.x should be doing the same thing here.